nixos/modules/services/web-apps/galene.nix

```diff
@@ -0,0 +1,130 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+let
+  cfg = config.services.galene;
+  defaultRecordingsRootPath = "${cfg.stateDir}/recordings";
+  defaultGroupsRootPath = "${cfg.stateDir}/groups";
+  defaultDataRootPath = "${cfg.stateDir}/data";
+in
+{
+  options = {
+    services.galene = {
+      enable = mkOption {
+        default = false;
+        type = types.bool;
+        description = "Enable Galene Service.";
+      };
+
+      stateDir = mkOption {
+        default = "/var/lib/galene";
+        type = types.str;
+        description = "Galene state directory.";
+      };
+
+      user = mkOption {
+        type = types.str;
+        default = "galene";
+        description = "User account under which galene runs.";
+      };
+
+      group = mkOption {
+        type = types.str;
+        default = "galene";
+        description = "Group under which galene runs.";
+      };
+
+      httpAddress = mkOption {
+        type = types.str;
+        default = "";
+        description = "HTTP listen address for galene.";
+      };
+
+      httpPort = mkOption {
+        type = types.int;
+        default = 8443;
+        description = "HTTP listen port.";
+      };
+
+      staticRootPath = mkOption {
+        type = types.str;
+        default = "${pkgs.galene.static}/static";
+        example = "/var/lib/galene/static";
+        description = "Web server directory.";
+      };
+
+      recordingsRootPath = mkOption {
+        type = types.str;
+        default = defaultRecordingsRootPath;
+        example = "/var/lib/galene/recordings";
+        description = "Recordings directory.";
+      };
+
+      dataRootPath = mkOption {
+        type = types.str;
+        default = defaultDataRootPath;
+        example = "/var/lib/galene/data";
+        description = "Data directory.";
+      };
+
+      groupsRootPath = mkOption {
+        type = types.str;
+        default = defaultGroupsRootPath;
+        example = "/var/lib/galene/groups";
+        description = "Web server directory.";
+      };
+    };
+  };
+
+  config = mkIf cfg.enable {
+    systemd.services.galene = {
```

Comment:
You should ask upstream to include a `galene.service` file in their repository. We could then make use of it.

Reply:
They just suggest something in their `README`: https://github.com/jech/galene

Reply:
If the `.service` file is included right in the repository as a file we can directly utilize it in this module.

Reply:
I asked upstream, i'll update if they choose to integrate the ```.service``` upstream.

Reply:
Upstream does not want to integrate a ```.service``` directly, so I'll keep my PR as such.

Reply:
You get full bonus points for trying though! Thanks.